amazon-s3/5.7/modules/ROOT/pages/amazon-s3-connector-examples.adoc

```diff
@@ -1,11 +1,6 @@
 = Amazon S3 Connector 5.7 Examples
 :page-aliases: connectors::amazon/amazon-s3-connector-examples.adoc
 
```

Comment:
Please put back the intro paragraph. You can reword it to mention examples instead of use cases. Something like this:

The following examples show how to:

Store...
Create